db/migrate/20160718152327_add_profiles_table.rb

```diff
@@ -0,0 +1,36 @@
+class AddProfilesTable < ActiveRecord::Migration
+  # rubocop:disable AbcSize, MethodLength
+  def up
+    create_table :profiles do |tbl|
+      tbl.integer :user_id, null: false
+      tbl.boolean :active, null: false, default: false
+      tbl.datetime :verified_at
+      tbl.datetime :activated_at
+      tbl.timestamps null: false
+
+      tbl.string :first_name
+      tbl.string :middle_name
+      tbl.string :last_name
+      tbl.string :address1
+      tbl.string :address2
+      tbl.string :city
+      tbl.string :state
+      tbl.string :zipcode
+      tbl.string :ssn
+      tbl.string :dob
+      tbl.string :phone
+      tbl.string :vendor
+    end
+
+    add_index :profiles, :user_id
+    # https://www.postgresql.org/docs/current/static/indexes-partial.html
+    execute('create unique index profiles_active_user_idx on profiles(user_id, active) where active=true')
```

Comment:
I assume you use `up`/`down` rather than `change` here because you have this custom SQL thing?